src/main/java/core/basesyntax/Test.java

```diff
@@ -0,0 +1,68 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+/**
+ * Feel free to remove this class and create your own.
+ */
+public class Test {
+    public static void main(String[] args) {
+        Random random = new Random();
+        Shape[] shapes = new Shape[random.nextInt(10)];
+        for (int i = 0; i < shapes.length; i++) {
```

Comment:
move the logic of creating random figure into a separate class, your main method is too large